BMS Workspace-git/bms-1227-fw/Phantom_drivers/drivers/source/Current_transducer.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Current_transducer.c
+ *
+ *  Created on: Nov 2, 2021
+ *      Author: Ryan Heo
+ */
+
+#include <Current_transducer.h>
+#include "adc.h"
+#include "stdlib.h"
+
+double getHVsensorVoltage()
+{
+    adcData_t adc_data;   // ADC data structure
+    adcData_t *adc_data_ptr = &adc_data;  // ADC data pointer
+     // declare variables
+
+    adcInit();   // Initialize the ADC module
+
+    adcStartConversion(adcREG1, 1U);  // Start ADC conversion
```

Comment:
What type of conversion? Explain why we need the conversion not what the function does here. In general, if we want to take a look at what a function does, we would go directly to it.

Reply:
- added comment explaining that adcStartConversion moves data from the registers at ADC module 1, group 1 to somewhere it can be accessed by adcGetData() then stored in a variable (adcOutputValue)